tests/Command/PlatformCreateTest.php

```diff
@@ -0,0 +1,746 @@
+<?php
+
+namespace EclipseGc\CommonConsole\Tests\Command;
+
+use Consolidation\Config\Config;
+use EclipseGc\CommonConsole\Command\PlatformCreate;
+use EclipseGc\CommonConsole\CommonConsoleEvents;
+use EclipseGc\CommonConsole\Event\GetPlatformTypeEvent;
+use EclipseGc\CommonConsole\Event\GetPlatformTypesEvent;
+use EclipseGc\CommonConsole\Platform\PlatformFactory;
+use EclipseGc\CommonConsole\Platform\PlatformStorage;
+use EclipseGc\CommonConsole\PlatformInterface;
+use EclipseGc\CommonConsole\QuestionFactory;
+use EclipseGc\CommonConsole\Tests\CommonConsoleTestBase;
+use EclipseGc\CommonConsole\Tests\Platform\FooPlatform;
+use Prophecy\Argument;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class PlatformCreateTest extends CommonConsoleTestBase {
+
+  public function testPlatformCreate() {
+    $dispatcher = $this->prophesize(EventDispatcherInterface::class);
+
+    $dispatcher->dispatch(CommonConsoleEvents::GET_PLATFORM_TYPES, Argument::type(GetPlatformTypesEvent::class))->will(function($arguments) {
+      /** @var GetPlatformTypesEvent $event */
+      $event = $arguments[1];
+      $event->addPlatformType('foo_platform');
+    });
+
+    $dispatcher->dispatch(CommonConsoleEvents::GET_PLATFORM_TYPE, Argument::type(GetPlatformTypeEvent::class))->will(function($arguments) {
+      /** @var GetPlatformTypeEvent $event */
+      $event = $arguments[1];
+      $event->addClass(FooPlatform::class);
+    });
+
+    $storage = $this->prophesize(PlatformStorage::class);
+    $storage->save(Argument::type(PlatformInterface::class))->will(function ($arguments) {
+      return $arguments[0];
+    })
+    ->shouldBeCalledOnce();
+
+    $factory = $this->prophesize(PlatformFactory::class);
+    $factory->getMockPlatformFromConfig(Argument::type(Config::class), Argument::type(PlatformStorage::class))->will(function($arguments) {
+      /** @var Config $config */
+      $config = $arguments[0];
+      /** @var PlatformStorage $storage */
+      $storage = $arguments[1];
+      return new FooPlatform($config->export(), $storage);
+    });
+
+    $questionFactory = new QuestionFactory($this->getContainer());
+
+    $command = new PlatformCreate($dispatcher->reveal(), $storage->reveal(), $factory->reveal(), $questionFactory);
+    $command->setApplication($this->getContainer()->get('common_console_application'));
+    $commandTester = new CommandTester($command);
+    $commandTester->setInputs([
+      'foo_platform',
+      'Foo Test',
+      'footest',
+    ]);
+    $rc = $commandTester->execute(['command' => $command->getName()]);
+
+    $this->assertEquals($this->getCharOutput($this->getExpectedOrdOutput()), $commandTester->getDisplay());
```

Comment:
I was playing around with this part and I was not able to figure it out what the heck is happening there. When I was testing this part I was using a different approach. 
`  $expected = $this->getExpectedDisplay();`  
`  $this->assertStringContainsString($expected[0], $commandTester->getDisplay(TRUE));`  
`  $this->assertStringContainsString($expected[1], $commandTester->getDisplay(TRUE));`  
The getExpectedDisplay() function returns an array. First value is the first part of the string from getDisplay() and the 2nd part is the second value within the array. Neither of those contain the problematic ' > foo_platform\n' string.

I know it is not 100% coverage but we could get rid off the getCharOutput() and getExpectedOrdOutput().